conda_smithy/lint_recipe.py

```diff
@@ -103,44 +103,44 @@ def lintify_meta_yaml(
     meta,
     recipe_dir=None,
     conda_forge=False,
-    is_rattler_build=False,
+    is_recipe_v2: bool = False,
```

Comment:
probably better to do something like
```suggestion
    recipe_version: int = 2,
```

Reply:
Agreed!